dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/icon/IconTest.java

```diff
@@ -87,12 +88,16 @@ protected void initTest() throws IOException {
   void shouldGetAllIconsWhenRequested() {
     Map<String, DefaultIcon> defaultIconMap = getAllDefaultIcons();
 
+    IconCriteria iconCriteria = new IconCriteria();
```

Comment:
You could also add one test similar to this one just using the default values from `iconCriteria`

Reply:
done